projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/__tests__/deploy-job-component.spec.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2023-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import DeployJobDialog, {
+  IDeployJobDialogProps
+} from '../components/DeployJob';
+import { render, fireEvent } from '@testing-library/react';
+import { jobData } from '../jobData';
+import { VdkOption } from '../vdkOptions/vdk_options';
+
+const defaultProps: IDeployJobDialogProps = {
+  jobName: 'test-name',
+  jobTeam: 'test-team',
+  jobPath: 'test-path'
+};
+
+describe('#constructor()', () => {
+  it('should return a new instance', () => {
+    const box = new DeployJobDialog(defaultProps);
+    expect(box).toBeInstanceOf(DeployJobDialog);
+  });
+});
+
+describe('#render()', () => {
+  it('should return contain job name input with placeholder equal to jobName from props', () => {
+    const box = new DeployJobDialog(defaultProps);
+    const component = render(box.render());
+    const input = component.getByPlaceholderText(defaultProps.jobName);
+    expect(input).toBe(component.getAllByLabelText('Job Name:')[0]);
+  });
+
+  it('should return contain job team input with placeholder equal to jobTeam from props', () => {
+    const box = new DeployJobDialog(defaultProps);
+    const component = render(box.render());
+    const input = component.getByPlaceholderText(defaultProps.jobTeam);
+    expect(input).toBe(component.getAllByLabelText('Job Team:')[0]);
+  });
+
+  it('should return contain job path input with placeholder equal to jobPath from props', () => {
+    const box = new DeployJobDialog(defaultProps);
+    const component = render(box.render());
+    const input = component.getByPlaceholderText(defaultProps.jobPath);
+    expect(input).toBe(component.getAllByLabelText('Job Path:')[0]);
+  });
+});
+
+describe('#onEnableClick', () => {
+  it('should put a flag for enabled in jobData', () => {
+    const box = new DeployJobDialog(defaultProps);
+    const component = render(box.render());
+    const enableCheckbox = component.getAllByLabelText(
+      'Enable - un-pause the job'
```

Comment:
I was thinking about adding those info elements, as well. But it is a separate issue and all the info elements will be added at once for all the components. I decided to put it as an separate issue since the previously implemented components does not have it and adding all info elements at once seems more correct.